sections/page.liquid

```diff
@@ -20,11 +20,11 @@
 
 <div class="color-{{ section.settings.color_scheme }} gradient">
   <div class="page-width page-width--narrow section-{{ section.id }}-padding">
-    <h2 class="page-title">
+    <h2 class="page-title {{ section.settings.heading_size }}">
       {%- if section.settings.page.title != blank -%}
         {{ section.settings.page.title | escape }}
       {%- else -%}
-        Page title
+        {{ 'sections.page.title' | t }}
```

Comment:
I wonder if we should offer the option to have none or overwrite it using a new setting for the section 🤷